src/test/java/com/microsoft/sqlserver/jdbc/preparedStatement/BatchExecutionWithBulkCopyTest.java

```diff
@@ -721,6 +721,45 @@ public void testNonSupportedColumns() throws Exception {
         }
     }
 
+    @Test
+    public void testReverseColumnOrder() throws Exception {
+        String valid = "insert into " + AbstractSQLGenerator.escapeIdentifier(tableName) + " (c2, c1) values "
+                + "(" + "?, " + "? " + ")";
+
+        try (Connection connection = PrepUtil.getConnection(connectionString + ";useBulkCopyForBatchInsert=true;");
+                SQLServerPreparedStatement pstmt = (SQLServerPreparedStatement) connection.prepareStatement(valid);
+                Statement stmt = (SQLServerStatement) connection.createStatement();) {
```

Comment:
Unneeded cast and unnecessary semicolon.

```suggestion
                Statement stmt = connection.createStatement()) {
```